generator/src/test/java/com/scottlogic/deg/generator/cucumber/longerThan.feature

```diff
@@ -0,0 +1,671 @@
+  And foo is anything but longer than 3
+  And foo is in set:
+  | 1 |
+  | "abc" |
+  | 2011-01-01T00:00:00.000 |
+  Then the following data should be generated:
+  | foo |
+  | null |
+  | 1    |
+  | 2011-01-01T00:00:00.000 |
+
+Scenario: Running a 'longerThan' request against non contradicting shorterThan should be succesful
+  Given foo is longer than 2
+  And foo is shorter than 5
+  And foo is in set:
+  | "a" |
+  | "aa" |
+  | "aaa" |
+  | "aaaa" |
+  | "aaaaaa" |
+  Then the following data should be generated:
+    | foo |
+    | null |
+    | "aaa" |
+    | "aaaa" |
+
+Scenario: Running a 'longerThan' request against non contradicting shorterThan should be succesful
+  Given foo is longer than 2
+  And foo is anything but shorter than 1
+  And foo is in set:
+    | "a" |
+    | "aa" |
+    | "aaa" |
+    | "aaaa" |
+  Then the following data should be generated:
+    | foo |
+    | null |
+    | "aaa" |
+    | "aaaa" |
+
+Scenario: Running a 'longerThan' request against non contradicting shorterThan should be succesful
+  Given foo is anything but longer than 5
+  And foo is shorter than 3
+  And foo is in set:
+    | "a" |
+    | "aa" |
+    | "aaa" |
+    | "aaaa" |
+    | "aaaaa" |
+  Then the following data should be generated:
+    | foo |
+    | null |
+    | "a"  |
+    |"aa" |
+
+Scenario: Running a 'longerThan' request against non contradicting shorterThan should be successful
+  Given foo is anything but longer than 3
+  And foo is anything but shorter than 1
+  And foo is in set:
+    | "a" |
+    | "aa" |
+    | "aaa" |
+    | "aaaa" |
+  Then the following data should be generated:
+    | foo |
+    | null |
+    | "a"  |
+    | "aa" |
+   | "aaa" |
+
+Scenario: Running a 'longerThan' request against contradicting shorterThan constraint should only generate numeric,temporal and null
+    Given foo is longer than 3
+    And foo is shorter than 2
+    And foo is in set:
+      | 1 |
+      | "abc" |
+      | 2011-01-01T00:00:00.000 |
+    Then the following data should be generated:
+      | foo |
+      | null |
+      | 1    |
+      | 2011-01-01T00:00:00.000 |
+Scenario: Running a 'longerThan' request against contradicting shorterThan constraint should only generate numeric,temporal and null
+  Given foo is anything but longer than 3
+  And foo is anything but shorter than 4
+  And foo is in set:
+    | 1 |
+    | "abc" |
+    | 2011-01-01T00:00:00.000 |
+  Then the following data should be generated:
+    | foo |
+    | null |
+    | 1  |
+    | 2011-01-01T00:00:00.000 |
+
+  @ignore #issue 487
+  Scenario: Running an 'longerThan' request alongside a non-contradicting aValid constraint should be successful
+    Given foo is longer than 12
+    And foo is a valid "ISIN"
+    And foo is in set:
+      | "US0000XVGZA3" |
+      | "US0378331005" |
+      | "22"           |
+    Then the following data should be generated:
+      | foo            |
+      | null           |
+      | "US0000XVGZA3" |
+      | "US0378331005" |
```

Comment:
I would not expect these two strings to be generated because they are of length 12. To make this test the correct behaviour I would change the first line to `Given foo is longer than 11`. 

I might also add a string that is of length 12 but is not a valid ISIN instead of "22".